autoload/codefmt/juliaformatter.vim

```diff
@@ -0,0 +1,75 @@
+" Copyright 2023 Google LLC
+"
+" Licensed under the Apache License, Version 2.0 (the "License");
+" you may not use this file except in compliance with the License.
+" You may obtain a copy of the License at
+"
+"     http://www.apache.org/licenses/LICENSE-2.0
+"
+" Unless required by applicable law or agreed to in writing, software
+" distributed under the License is distributed on an "AS IS" BASIS,
+" WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+" See the License for the specific language governing permissions and
+" limitations under the License.
+
+let s:plugin = maktaba#plugin#Get('codefmt')
+
+let s:checkedInstall = 0
+
+""
+" @private
+" Formatter: JuliaFormatter
+function! codefmt#juliaformatter#GetFormatter() abort
+  let l:installer =
+        \ maktaba#path#Join([expand('<sfile>:p:h:h'), 'bin', 'julia', 'install'])
```

Comment:
What about `maktaba#path#Join([s:plugin.location, 'bin', 'julia', 'install'])` here, to make it a little more robust about where this particular sfile sits?

Reply:
Oh cool, I didn't realize `plugin.location` was a thing.  That's definitely better.